gateleen-security/src/main/java/org/swisspush/gateleen/security/authorization/ConfigurationResource.java

```diff
@@ -0,0 +1,24 @@
+package org.swisspush.gateleen.security.authorization;
+
+import org.swisspush.gateleen.validation.ValidationException;
+import io.vertx.core.buffer.Buffer;
+
+/**
+ * Interface declaring the method to be commonly used for role related resource configuration
+ */
+interface ConfigurationResource {
```

Comment:
consider making it public so other implementations (from other packages) are possible

Reply:
Here I disagree with @oli-h. See my comment in the _Authorizer_ class about using _ConfigurationResourceManager_. I would not further propagate the usage of this _ConfigurationResource_ interface when we already have a solution for this in gateleen.

Reply:
I'll keep it local therefore. Should be once rewritten to use common implementation